src/dal/Jalasoft.TeamUp.Resumes.DAL/ResumesRepository.cs

```diff
@@ -82,13 +82,19 @@ public class ResumesRepository : IResumesRepository
 
         public Resume GetResume(Guid id)
         {
-            Resume result = Resumes.FirstOrDefault(p => Equals(p.Id, id));
+            Resume result = resumes.FirstOrDefault(p => Equals(p.Id, id));
             return result;
         }
 
         public IEnumerable<Resume> GetResumes()
         {
-            return Resumes;
+            return resumes;
+        }
+
+        public Resume PostResumes(Resume resume)
+        {
+            resumes = new List<Resume>(resumes) { resume }.ToArray();
```

Comment:
```suggestion
            resumes.Add(resume);
```

Reply:
Sorry Cristian, resumes is an array and not have the Add function